### DISCUSSION TITLE
【请求扩散】警告各位

### DISCUSSION THREAD

**Discussion by Tragedysaver1024** [share121/inter-knot#588] [其他]
[![pAStu2d.md.jpg](https://s21.ax1x.com/2024/08/10/pAStu2d.md.jpg)](https://imgse.com/i/pAStu2d)
如果半夜（尤其是2:00～6:00这个区间）上厕所或者在其他什么地方的时候发现手机变成这样
不要抬头！
不要抬头！！
不要抬头！！！
这是你这台手机能为你做到的最后一件事！！！！

**Comment by GUILAISHIXV** [comment]
这个委托我接了

**Comment by themagicmoon** [comment]
我……不知道说什么，2点39看到的😫

**Comment by 290678** [comment]
没有手机只有电脑怎么办
